deepmd/loss/ener.py

```diff
@@ -384,4 +384,169 @@ def print_on_training(self,
         return print_str   
 
 
+class EnerForcesMaskLoss(Loss):
+    """
+    EnerForcesMaskLoss is a concise version of the loss function for DP models.
+    Forces Difference with be multiplied by a `mask4force_matrix: [#frame, #atoms * 3]`.
+    User can specifiy on the forces on which atoms to be fitted by setting `mask4force_matrix`.
 
+    Parameters
+    ----------
+
+    """
+    def __init__(self,
+                 starter_learning_rate : float, 
+                  start_pref_e : float = 0.02,
+                  limit_pref_e : float = 1.00,
+                  start_pref_f : float = 1000,
+                  limit_pref_f : float = 1.00, 
+                  **kwargs)-> None :
+        model = kwargs.get('model', None)
+        if model is not None:
+            self.ntypes = model.ntypes
+            self.type_map = model.type_map
+        else:
+            self.type_map = None
+        self.starter_learning_rate = starter_learning_rate
+        self.start_pref_e = start_pref_e
+        self.limit_pref_e = limit_pref_e
+        self.start_pref_f = start_pref_f
+        self.limit_pref_f = limit_pref_f
+        self.has_e = (self.start_pref_e != 0.0 or self.limit_pref_e != 0.0)
+        self.has_f = (self.start_pref_f != 0.0 or self.limit_pref_f != 0.0)
+    
+        self.debug_op = []
+        
+        # data required
+        add_data_requirement('energy', 1, atomic=False, must=False, high_prec=True)
+        add_data_requirement('force',  3, atomic=True,  must=False, high_prec=False)
+        add_data_requirement('mask_matrix', 1, atomic=True, must=True, high_prec=False)
+        add_data_requirement('mask_matrix4force', 3, atomic=True, must=False, high_prec=False)
```

Comment:
The default "default" value is zero. For `mask_matrix4force`, should the default value set to one?

Reply:
Yes, I will make the default value to be 1.0 in next commit.